pkg/deploy/defaults.go

```diff
@@ -71,12 +71,28 @@ const (
 	OldDefaultPostgresUpstreamImageToDetect = "centos/postgresql-96-centos7:9.6"
 
 	// ConsoleLink default
-	DefaultConsoleLinkName        = "che"
-	DefaultConsoleLinkImage       = "/dashboard/assets/branding/loader.svg"
-	DefaultConsoleLinkDisplayName = "Eclipse Che"
-	DefaultConsoleLinkSection     = "Red Hat Applications"
+	DefaultConsoleLinkSection              = "Red Hat Applications"
+	DefaultConsoleLinkImage                = "/dashboard/assets/branding/loader.svg"
+	defaultConsoleLinkCheName              = "che"
+	defaultConsoleLinkCodeReadyName        = "codeready"
+	defaultConsoleLinkCheDisplayName       = "Eclipse Che"
+	defaultConsoleLinkCodeReadyDisplayName = "CodeReady Workspaces"
```

Comment:
-1, for a few reasons...

a) "CodeReady" is a brand, not a product, so should NOT be used here. Options are "Red Hat CodeReady Workspaces", "CodeReady Workspaces", "Workspaces" or if an internal variable not exposed to the end user in the UI/UX, then CRW is ok. But...

b) the existing convention used in https://github.com/eclipse/che-operator/blob/master/pkg/deploy/defaults.go#L40-L41 to differentiate upstream/downstream versions of variables is with "Upstream" prefix on the variable name, so we could keep using that, eg., `defaultConsoleLinkUpstreamName` and `defaultConsoleLinkName`

where does the `ConsoleLinkName` appear? if it's UI, it shouldn't be "codeready" but rather "workspaces" or "Workspaces" as that's the product's shortname. "CodeReady" includes Studio, Builder, Containers, and more, so we can't just slap that in Workspaces and OCP and assume people will think we mean the Che-based product.

Reply:
cc @slemeur

Reply:
@slemeur Could you provide some guidance here ?

Reply:
Related to comment b, it's right that the most general case for variables is to have the *raw property name* for CRW, and *property name + "upstream"* for Eclipse Che. So to increase consistency we could use this naming scheme here as well.

As for the consoleLinkName, I'm don't think it is displayed to the user, and changing it might break the OS 4.2 integration if they would be searching for the ConsoleLink by name.

In fact looking into the OpenShift 4.2 console, it is clear that **we should not** change the ConsoleLinkName:

https://github.com/openshift/console/blob/f5a2a8b0358398748f301af0c11bd74ac33badce/frontend/packages/dev-console/src/components/topology/topology-utils.ts#L27